CHANGELOG.md

```diff
@@ -4,6 +4,18 @@
 
 All notable changes to this project will be documented in this file.
 
+#### v2.2.0 - 2018-01-21
+  - Migrated to TypeScript
+  - Added full support for the Orders API
+  - Add tests for the Orders API and base classes and a few regression tests to guard against removing deprecated functionality too soon
+  - Added Orders API examples
+  - Add browser support (decoupled `cacert.pem`, remove node.js only dependencies)
```

Comment:
@mattijsbliek noted that we should not remove the Node.js only dependencies, since it's a Node API client. 

Doing so will allow users to use this in the browser, which means they will expose their API keys publicly. This is not something we want.

Reply:
Maybe we should even throw an error when `typeof window !== 'undefined'` upon API client initialization. Our technical support department often gets questions because developers try to use the API client in a browser environment. We can prevent support calls by throwing an `UnsupportedEnvironmentException` :)

Reply:
I'll double check what parts are added to the browser build by Meteor.js. I had to remove a couple of node.js dependencies to make it work on Reaction Commerce.
Maybe those dependencies would only have to be removed from the entry file, but the way Meteor handles it all is magic to me right now.

Reply:
Meteor.js automatically splits files between browser and server, that is what I can tell from what I've seen in the final production build, but I wonder what files it picks. It might be just the entry file that ends up in the browser build.

Reply:
I'm okay with removing these dependencies, if we don't need them.
In that case we should definitely throw an error when the global `window` reference is available, to prevent people from using it within in the browser.

Reply:
Ahh, that might have happened in older commonjs only builds. In those builds everything ends up in one large file and there is no way to exclude code in browser builds (tree shaking). Reaction Commerce has tree shaking enabled by default, which means that it should now use `jsnext:main` or `module` as an entry point in `package.json` and be able to prune whatever's not imported in the [`client` folder](https://github.com/mollie/mollie-reaction-commerce/tree/aa1800138958bfb498609ab3017c1635893bed9d/client) of a Reaction Commerce plugin.

Reply:
Just a thought! Have yet to confirm.

Reply:
ESM build no longer looks like to be an issue with Reaction Commerce, nice!